.config/gitpod/Dockerfile

```diff
@@ -1,9 +1,8 @@
-FROM ubuntu:jammy
+# First stage: Install build-time dependencies
```

Comment:
Congratulations on this contribution! Thank you very much. 

Have you carried out tests using the `make docker-tests` command, in particular ? 

Also, don't hesitate to run `gp validate` to ensure it works correctly on Gitpod

Reply:
Thank you for the congratulations, it was a pleasure to contribute to this project.

Throughout the process of fixing and writing my code, I indeed used the commands `make tests` and `make docker-tests` to ensure that my modifications did not generate any other errors in the project during execution.

Regarding the `gp validate` command, I used it in conjunction with the `make reinitization` command to avoid keeping the same mount point during my tests.

Hoping to have been helpful to this project,

Best regards.